Makefile

```diff
@@ -1,4 +1,6 @@
 example-client:
-	go run *.go generate -f testdata/pets.yaml | goimports | gofmt > example/client/client.go
+	go run *.go generate client -f testdata/pets.yaml | goimports | gofmt > example/client/client.go
+example-server:
+	go run *.go generate server -f testdata/pets.yaml | goimports | gofmt > example/server/server.go
```

Comment:
i believe there are no need in gofmt after goimports. of goimports before gofmt =)